packages/synapse-react-client/src/components/EntityViewScopeEditor/EntityViewScopeEditor.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useMemo } from 'react'
+import EntityHeaderTable from '../EntityHeaderTable'
+import { EntityType, ReferenceList } from '@sage-bionetworks/synapse-types'
+import { VersionSelectionType } from '../EntityFinder/VersionSelectionType'
+import { FinderScope } from '../EntityFinder/tree/EntityTree'
+import { EntityTypeGroup } from '../../utils/functions/EntityTypeUtils'
+import { Typography } from '@mui/material'
+
+const DEFAULT_SELECTABLE_TYPES = EntityTypeGroup.CONTAINER
+const PROJECT_VIEW_SELECTABLE_TYPES = [EntityType.PROJECT]
+
+export type EntityViewScopeEditorProps = {
+  scopeIds: string[]
+  onChange: (scopeIds: string[]) => void
+  isProjectView?: boolean
+}
+
+export default function EntityViewScopeEditor(
+  props: EntityViewScopeEditorProps,
+) {
+  const { scopeIds, isProjectView = false } = props
```

Comment:
Could `onChange` be destructured here too? Just curious if there are situations where it's best to access a value directly from the `props` object

Reply:
Just an inconsistency that I will fix. The only times I think it matters are
1. If you destructure some props and spread the rest e.g. ` const { foo, bar, ...rest } = props` since the destructured props affect the value of `rest`
2. Where Playwright somehow tracks which values you destructure in the function signature (which still doesn't make a lot of sense to me 😅)